### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by oovillagran**
Based on a database diagram, we interpreted it and implemented the SQL needed to build a database.

**Project requirements:**

- Take a look at the following database diagram for a clinic:
image
- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    invoice_id INT,
+    treatment_id INT,
+    CONSTRAINT FK_INVOICE_ID FOREIGN KEY (invoice_id) REFERENCES invoices(id),
+    CONSTRAINT FK_TREATMENT_ID FOREIGN KEY (treatment_id) REFERENCES treatments(id)
+);
+
+CREATE INDEX idx_patient_id ON medical_histories (patient_id);
+CREATE INDEX idx_medical_history_id ON medical_histories_treatments (medical_history_id);
+CREATE INDEX idx_treatments_id ON medical_histories_treatments (treatments_id);
+CREATE INDEX idx_invoices_id ON invoices (medical_history_id);
```
Comment (laiifuu) [inline schema_based_on_diagram.sql:53]: Nice job creating the indexes for the foreign keys :+1: 
However, as mentioned in the project requirements, ALL the foreign keys should have an index.
Indexes for the foreign keys in the table `invoice_items` (treatment_id, invoice_id) are missing.
Kindly add them to the file.

**Review by laiifuu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: 

## Hi team :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
so-close

## Highlights
- [x] All tables in the diagram were created :heavy_check_mark: 
- [x] The "join table" for the many-to-many relationship is created :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (oovillagran) [reply schema_based_on_diagram.sql:53]: Done! Thanks for your feedback @laiifuu.

**Review by t-yanick — APPROVED** [review]
# STATUS: MILESTONE APPROVED :1st_place_medal: 

Hi Team, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  ALL foreign keys have indexes

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oovillagran** [comment]
Thanks @t-yanick.

**Pull request merged**